TODO.txt

```diff
@@ -1,72 +1,4 @@
-PG2:
-
- * link for translation sources
- * show that newer content is available for missed slugs (fallback to v1)
-
-
-
-3.1 Launch:
-
-* add MIT/CC license link to footer
-
 * asciidoc parsing
   * convert URLs to links
   * nested dlists (eg. diff.mnemonicprefix in diff-config.txt)
   * dlist dd with nested ulist with continuations (eg. git-blame -L)
```

Comment:
I don't think we'd want to actively work on anything related to asciidoc. These days the asciidoctor project is spun off into its own thing, and we'll take whatever it renders (and some of these, like the nested dlists, seem to work already).